configurations/perf/regression-nosqlbench-aws-r5-2xlarge.yaml

```diff
@@ -0,0 +1,3 @@
+instance_type_db: 'r5.2xlarge'
```

Comment:
@KnifeyMoloko did you test it with this configuration?
Didn't you need to add a parameter for using EBS volume?

In EBS perf regression tests where we use r5.2xlarge we used:
data_volume_disk_num: 2
data_volume_disk_type: "gp3"
data_volume_disk_size: 2000
data_volume_disk_iops: 16000 # depend on type iops could be 100-16000 for io2|io3 and 3000-16000 for gp3

What was the config the used in the blog?

Reply:
@roydahan The blog example used a single 3TB gp2 EBS with 10000 disk_iops. I'm in the process of testing the r5 configuration. I also added the specific data volume definition to the conif yaml.